.github/actions/image-builder/action.yml

```
@@ -42,6 +42,10 @@ inputs:
     description: Prevent actual triggering the image builder
     required: false
     default: false
+  use-go-internal-modules:
```

```suggestion
  use-go-sap-internal-modules:
```
maybe we should add that it is sap internal module?

---

Sounds reasonable